bats/fb-verify-packages.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:ts=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+@test "verify no old SCL packages" {
+  run tSetOSVersion
+  run FOREMAN_VERSION=$(tForemanVersion)
+
+  if tIsRedHatCompatible; then
+    if [[ $(echo "$FOREMAN_VERSION >= 1.20" | bc) == 1 ]]; then
+      if [ `yum list installed | grep -q -E 'rh-ruby24|tfm-ror51' | wc -l` > 0 ]; then
```

Comment:
Ah, I now see why the counting is needed. Perhaps we should use `grep -c` instead?